edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
```

Comment:
Use prepared statements for both selecting and updating book records to avoid SQL injection vulnerabilities.
Create an abstraction logic to separate concerns following MVC patterns.